services/userservice/userservice.go

```diff
@@ -48,3 +49,12 @@ func AuthenticateUserCredentials(emailAddress string, password string) error {
 
 	return nil
 }
+
+func GetIdForUserWithEmailAddress(emailAddress string) (models.UserId, error) {
+	number, err := databaseutil.GetUserIdFromUserWithEmailAddress(emailAddress)
```

Comment:
i would go with `userIdValue` as opposed to `number`

Reply:
used `userIdAsInt`